server/src/test/java/org/apache/iotdb/db/metadata/MManagerBasicTest.java

```diff
@@ -360,4 +363,26 @@ public void testMaximalSeriesNumberAmongStorageGroup() throws MetadataErrorExcep
     manager.deletePaths(Collections.singletonList(new Path("root.laptop.d1.s2")));
     assertEquals(1, manager.getMaximalSeriesNumberAmongStorageGroups());
   }
+
+  @Test
+  public void testGetStorageGroupNameByAutoLevel() {
+    MManager manager = MManager.getInstance();
+    int level = IoTDBDescriptor.getInstance().getConfig().getAutoStorageGroupLevel();
+    try {
+      Assert.assertEquals("root.laptop", manager.getStorageGroupNameByAutoLevel("root.laptop.d1.s1", level));
+    } catch(PathErrorException e) {
+      e.printStackTrace();
+      fail(e.getMessage());
+    }
+    try {
+      manager.getStorageGroupNameByAutoLevel("root1.laptop.d1.s1", level);
+    } catch(PathErrorException e) {
+      Assert.assertEquals("Timeseries root1.laptop.d1.s1 is not right.", e.getMessage());
```

Comment:
This is not a good way to test exceptions because the tests still pass if there is just no exception, which is also incorrect.
I would usually write code like:
```
boolean caught = false;
try {
  doSomething();
} catch (WantedException e) {
  // verify message
  caught = true;
}
assertTrue(caught);
```
I know this is not the best way, neither, but it can assure the exception is risen and caught. If you find a better way, please inform me.